AudiusProject/audius-protocol
# Audio analysis: mediorum changes

### Description
- Add cols to uploads. Analysis results are in 1 column so it should be extensible
- Allow 1 minute to analyze audio in mediorum after transcoding completes
- Analysis failures/timeouts/errors do not block the upload. It gets marked as done regardless of analysis result
- Add endpoint to retry failed or missing audio analyses
- Installing Essentia in the mediorum Dockerfile was a huge pain and the Essentia docker images are very outdated and different than the Essentia Python I used in my test script so I abandoned this library in favor of https://github.com/mixxxdj/libkeyfinder/tree/main (per Marcus' recommendation) for key detection and https://aubio.org/ for BPM detection. We can always replace these tools later if we find something better.

### How Has This Been Tested?
- Run locally via audius-compose up mediorum
- Verify analysis runs
<img width="1728" alt="image" src="https://github.com/AudiusProject/audius-protocol/assets/25782182/7ad4df67-517c-49ad-a5ad-135e31c36a66">
- Verify server changes
- Make code change to sleep for 2 mins when handling audio analysis job, verify 1 min timeout applies and upload completes after 1 min
- Verify prod dockerfile builds
